Source/Core/VideoCommon/UberShaderVertex.cpp

```diff
@@ -462,14 +462,14 @@ static void GenVertexShaderTexGens(APIType api_type, u32 num_texgen, ShaderCode&
             "      }}\n"
             "    }}\n"
             "    break;\n\n");
-  out.Write("  case {}u: // XF_TEXGEN_COLOR_STRGBC0\n", XF_TEXGEN_COLOR_STRGBC0);
+  out.Write("  case {:s}:\n", TexGenType::Color0);
   out.Write("    output_tex.xyz = float3(o.colors_0.x, o.colors_0.y, 1.0);\n"
             "    break;\n\n");
-  out.Write("  case {}u: // XF_TEXGEN_COLOR_STRGBC1\n", XF_TEXGEN_COLOR_STRGBC1);
+  out.Write("  case {:s}:\n", TexGenType::Color1);
   out.Write("    output_tex.xyz = float3(o.colors_1.x, o.colors_1.y, 1.0);\n"
             "    break;\n\n");
-  out.Write("  default:  // Also XF_TEXGEN_REGULAR\n"
-            "    {{\n");
+  out.Write("  default:  // Also {}\n", TexGenType::Regular);
```

Comment:
` out.Write("  default:  // Also {:s}\n", TexGenType::Regular);`

?

Reply:
Semi-intentional, as it's showing up in a comment so `default:  // Also Regular (0)` looks better than `default:  // Also 0x0u /* Regular */`.  But perhaps just adding a `case 0x0u /* Regular */:` before the default would more clearly express the intent.